Cubical/Structures/Group/MorphismProperties.agda

```diff
@@ -28,63 +28,84 @@ private
   variable
     ℓ ℓ' ℓ'' : Level
 
+infixr 31 _□_
+
+open Group
+open GroupHom
+open IsMonoid
+open IsSemigroup
+open IsGroup
```

Comment:
Do you need to open all of these?

Reply:
For this to run yes. Again, I could remove them if you want me to but then things get longer later.

Reply:
I thought Group would open IsMonoid, IsSemigroup and IsGroup automatically for you...

Reply:
Look at this: https://github.com/agda/cubical/blob/master/Cubical/Structures/Group/Base.agda#L48

They should recursively open things for you. Did you try to only open Group and GroupHom?

Reply:
Look at this: https://github.com/agda/cubical/blob/master/Cubical/Structures/Group/Base.agda#L48

They should recursively open things for you. Did you try to only open Group and GroupHom?